config/ppo/WallJump_curriculum.yaml

```diff
@@ -49,20 +49,88 @@ behaviors:
     time_horizon: 128
     summary_freq: 20000
     threaded: true
-
-curriculum:
-  BigWallJump:
-    measure: progress
-    thresholds: [0.1, 0.3, 0.5]
-    min_lesson_length: 100
-    signal_smoothing: true
-    parameters:
-      big_wall_min_height: [0.0, 4.0, 6.0, 8.0]
-      big_wall_max_height: [4.0, 7.0, 8.0, 8.0]
-  SmallWallJump:
-    measure: progress
-    thresholds: [0.1, 0.3, 0.5]
-    min_lesson_length: 100
-    signal_smoothing: true
-    parameters:
-      small_wall_height: [1.5, 2.0, 2.5, 4.0]
+environment_parameters:
+  big_wall_min_height:
```

Comment:
Can you big_wall_min_height and big_wall_max_height into a single parameter with uniform sampling? Seems like that would be more concise, provide an example of how to do both sampling and fixed values, and simply the C# example code.

Reply:
Yes, coming soon